grml-debootstrap

```diff
@@ -26,7 +25,6 @@ set -e
 set -E
 set -o pipefail
 trap "error_handler" ERR
-export -f "error_handler"
```

Comment:
is removing the export of the error_handler from `grml-debootstrap` and addition of it to chroot script related to the environment cleanup, right?

Reply:
Ah, you mentioned this in a comment as I just noticed:

> If we use env -i then we can no longer export shell functions. So export -f "error_handler" had to be removed.

We should squash all the relevant commits into one single commit and provide that useful information you collected into the commit message :)

Do you want me to give this a try or would you prefer to try on your own?

Reply:
> Do you want me to give this a try

Yes, please.